__frzr-deploy

```diff
@@ -289,20 +289,23 @@ main() {
 
 	rm -rf /var/lib/pacman # undo frzr-unlock
 
-	echo "deployment complete; restart to boot into ${NAME}"
 
 	# Check if there are migrations available
 	if compgen -G "${SUBVOL}"/usr/lib/frzr.d/*.migration > /dev/null ; then
 		for m in "${SUBVOL}"/usr/lib/frzr.d/*.migration ;
 		do
-			echo "Applying migration $(basename $m)"
 			unset -f post_install
 			. $m
-			post_install "${MOUNT_PATH}" "${SUBVOL}"
+			if [ "$(type -t foo)" == function ] ; then
+			  echo "Applying post install migration $(basename $m)"
+        post_install "${MOUNT_PATH}" "${SUBVOL}"
```

Comment:
indentation looks off. mixed spaces and tabs.

Reply:
Should be fixed now